### PR TITLE
Individual cops can be excluded for files

### PR DESCRIPTION
**Pull request by brittballard**
* Pass the file's path into ProcessedSource initializer
* This path is later used to determine if a specific Cop is applicable
* https://github.com/thoughtbot/hound/issues/522

### CODE REVIEW

**Comment by brittballard** [comment]
This is the line that does the check for the specific file: https://github.com/bbatsov/rubocop/blob/master/lib/rubocop/cop/commissioner.rb#L56.

**Comment by brittballard** [comment]
Do you guys really want me to shorten those strings in the specs?

File: spec/models/style_guide/ruby_spec.rb [hunk 1]
```diff
@@ -420,6 +420,23 @@ def initialize(name: , age: )
...
+          },
+          "HashSyntax" => {
+            "Exclude" => ["lib/**/*"]
+          }
+        }
+
+        violations = violations_with_config(config)
+
+        expect(violations).to eq [
+          "Prefer single-quoted strings when you don't need string interpolation or special symbols."
```
Comment (gylaz) [inline spec/models/style_guide/ruby_spec.rb]: Yeah, I'd say just break it up with a `+`. We can't preach what we don't practice.

**Comment by gylaz** [comment]
Ready to merge once the hound comments are handled!

**Pull request closed without merging**